src/lore_session.rs

```diff
@@ -74,16 +76,16 @@ impl LoreSession {
     fn process_patches(&mut self, patch_feed: PatchFeed) -> Vec<String> {
         let mut processed_patches_ids: Vec<String> = Vec::new();
 
-        for mut patch in patch_feed.get_patches() {
+        for mut patch in patch_feed.patches().clone() {
```

Comment:
To tell you the truth, I spent quite a while thinking about adding the getter for the `PatchFeed` type to the detriment of having to clone here. At first, I went straight to redoing the old `get_patch()` that returned an owned type `Vec<Patch>`, but with the name `patches()`.

However, I realized that `PatchFeed` is instantiated and soon dropped when the variable `patch_feed` of `process_n_representative_patches` goes out of scope. Combining the fact that, per request, we fill 200 patches (`Patch` not being a huge struct, itself), then soon drop it, I reverted to your approach :rofl: